doc/geany.txt

```diff
@@ -2527,6 +2527,8 @@ show_editor_scrollbars            Whether to display scrollbars. If set to     t
 indent_hard_tab_width             The size of a tab character. Don't change    8           immediately
                                   it unless you really need to; use the
                                   indentation settings instead.
+editor_ime_inline                 Whether to display input method editor's     true        to new
+                                  candidate window inline                                  documents
```

Comment:
The setting is in `editor_apply_update_prefs` which comments that it is non-document prefs, so does this really apply to new documents only?